src/transformer/events/mod_quiz/question_answered/numerical.php

```diff
@@ -49,7 +49,7 @@ function numerical(array $config, \stdClass $event, \stdClass $questionattempt,
         ],
         'timestamp' => utils\get_event_timestamp($event),
         'result' => [
-            'response' => $questionattempt->responsesummary,
+            'response' => ($questionattempt->responsesummary ? $questionattempt->responsesummary:''),
```

Comment:
This could be simplified a bit by using `?:`.

```suggestion
            'response' => ($questionattempt->responsesummary ?: ''),
```

Reply:
OK - made this change